pgsync/sync.py

```diff
@@ -1285,15 +1295,17 @@ def async_poll_db(self) -> None:
 
         while self.conn.notifies:
             notification: t.AnyStr = self.conn.notifies.pop(0)
+            self.count['notifications'] += 1
             if notification.channel == self.database:
                 payload = json.loads(notification.payload)
-                if self.index in payload["indices"]:
```

Comment:
why is this removed?
seems to be needed and fix as per [fix](https://github.com/tolitius/pgsync/commit/de812696d9346e4a5a5a744fa50f9696daddcbef) in the upstream repo we don't yet have

Reply:
this condition moved into should_skip_event